### PR TITLE
Cisoreview release 4.3-dont merge

### PR DESCRIPTION
**Pull request by kimetro**

### CODE REVIEW

File: assets/ca-operator/ca-operator-deployment.yaml [hunk 1]
```diff
@@ -5,16 +5,36 @@ metadata:
...
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - patch
+- apiGroups: ["extensions", "apps"]
+  resources:
+    - configmaps
```
Comment (relyt0925) [inline assets/ca-operator/ca-operator-deployment.yaml]: deployment

File: assets/ca-operator/ca-operator-deployment.yaml [hunk 2]
```diff
@@ -5,16 +5,36 @@ metadata:
...
+  verbs:
+    - get
+    - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
   name: ca-operator
+  namespace:
```
Comment (relyt0925) [inline assets/ca-operator/ca-operator-deployment.yaml]: no namespace field here

File: assets/ca-operator/ca-operator-deployment.yaml [hunk 3]
```diff
@@ -5,16 +5,35 @@ metadata:
...
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - patch
+- apiGroups: ["extensions", "apps"]
+  resources:
+  - deployment
```
Comment (relyt0925) [inline assets/ca-operator/ca-operator-deployment.yaml]: deployments instead of deployment

**Review by relyt0925 — CHANGES_REQUESTED** [review]
this should be closed

**Comment by kimetro** [comment]
Another PR already addressed the issue

**Pull request closed without merging**